packages/xchain-bitcoin/src/client.ts

```diff
@@ -97,16 +91,15 @@ class Client implements BitcoinClient, XChainClient {
    * Set/update a new phrase.
    *
    * @param {string} phrase A new phrase.
-   * @returns {Address} The address from the given phrase
+   * @returns {Address} The first address from the given phrase
    *
    * @throws {"Invalid phrase"}
    * Thrown if the given phase is invalid.
    */
   setPhrase = (phrase: string): Address => {
     if (validatePhrase(phrase)) {
       this.phrase = phrase
-      const address = this.getAddress()
-      return address
+      return this.getAddress(0)
```

Comment:
We can't expect `0` in cases users will use another index. Put `index: number` as optional parameter (as similar as you done it in Binance client)

Reply:
ok, it feels slightly dirty to have a setPhrase() return any address at all, but i'll enforce setNetwork(phrase:string, index?: number) in the XChainClient interface so it's consistent everywhere